ui/src/components/content/catalog/components/ServiceDetail.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import React from 'react';
+import { Descriptions, Divider, Tag } from 'antd';
+import { CloudUploadOutlined, InfoCircleOutlined } from '@ant-design/icons';
+import { OclDetailVo } from '../../../../xpanse-api/generated';
+import { get, map } from 'lodash';
+
+function ServiceDetail({
+    serviceDetails,
+    serviceRegions,
+}: {
+    serviceDetails: OclDetailVo;
+    serviceRegions: string[];
+}): JSX.Element {
+    return (
+        <>
+            <div className={'catalog-detail-class'}>
+                <h3>
+                    <CloudUploadOutlined />
+                    &nbsp;Available Regions
+                </h3>
+                {map(serviceRegions, (r) => (
+                    <Tag color='default'>{r}</Tag>
+                ))}
+                <Divider />
+            </div>
+            <h3>
+                <InfoCircleOutlined />
+                &nbsp;Basic Information
+            </h3>
+            <Descriptions bordered column={1}>
+                <Descriptions.Item label='Property' labelStyle={{ width: '230px' }}>
+                    Information
+                </Descriptions.Item>
+                <Descriptions.Item label='Category'>{get(serviceDetails, 'category')}</Descriptions.Item>
+                <Descriptions.Item label='Provider'>
+                    {get(serviceDetails, 'cloudServiceProvider.name')}
```

Comment:
is it possible to prefer access the value directly from the object instead of using lodash. We have proper classes generated. So accessing it directly is more cleaner way and there is no need to pass the string which can cause errors.

Reply:
The value directly from the object, if not obtained, the page will report an error, so  use get instead of the triadic operator.

Reply:
triadic operator is better in this case. It is more explicit on what we are trying to do.